### DISCUSSION TITLE
New changes break the design

### DISCUSSION THREAD

**Discussion by a-asharapova** [GitbookIO/community#509] [2. Published content]
### Ask a question, start a conversation, or make an annoucement

Hello everyone! 

I generally like the new design, but unfortunately, it breaks a few important elements for us:
1. I cannot find how to change the background color of the documentation. It was white before, and now it's grey.
2. The header shows only three links, even though in the old design it was showing four.

Please let me know how to solve these issues. Thank you!

**Comment by SamyPesse** [comment]
> I cannot find how to change the background color of the documentation. It was white before, and now it's grey.

You can change the background from the customization settings:

<img width="610" alt="Screenshot 2024-03-11 at 11 49 43" src="https://github.com/GitbookIO/community/assets/845425/fa7a7067-5928-4850-8506-1b1e422e67f8">

> The header shows only three links, even though in the old design it was showing four.

Can you share the links to your documentation? To adapt to different screen sizes, it might not show all links at all time.

**Comment by a-asharapova** [comment]
Thank you @SamyPesse. The solution for question 1 worked for us. Here is the link to our docs: https://docs.myparallels.com/parallels-daas-administrators-guide

**Comment by a-asharapova** [comment]
Update: in another document, https://docs.myparallels.com/parallels-browser-isolation-administrators-guide, only two of four header links are shown.
